examples/crd_one_var.R

```diff
@@ -0,0 +1,138 @@
+#-----------------------------------------------------------------------------#
+
+# Box plot of the effect on virus on yield.
+dev.new()
+#-----------------------------------------------------------------------------#
+boxplot(yield ~ virus, data = my.data, main = "Effect of virus on yield",
+        xlab = "virus", ylab = "yield (?)")
+#-----------------------------------------------------------------------------#
+
+# Plot two standard fit plots: residuals vs predicted, Normal Q-Q plot of the
+# residuals.
+dev.new()
+#-----------------------------------------------------------------------------#
+par(mfrow = c(2, 1), oma = c(0, 0, 2, 0))  # plots as subplots of single graph
+plot(model, which = c(1, 2))
+#-----------------------------------------------------------------------------#
+
+# Make sure the residuals are normal (this can also be seen in the Q-Q plot).
+cat('Shapiro-Wilk Normality Test\n')
+sep(79)
+#-----------------------------------------------------------------------------#
+shapiro.test(residuals(model))
+#-----------------------------------------------------------------------------#
+sep(79)
+
+# Run Levene's Test.
+cat("Levene's Test\n")
+sep(79)
+#-----------------------------------------------------------------------------#
+leveneTest(yield ~ virus, data = my.data)
+#-----------------------------------------------------------------------------#
+sep(79)
+
+#------------------------------------------------------------------------------#
+sep(79)
+
+# Print the ANOVA table of the fit. The user should will have to note the
+# significant factors. In this case the single virus factor is significant.
+cat('ANOVA Table\n')
+sep(79)
+#-----------------------------------------------------------------------------#
+anova(model)
+#-----------------------------------------------------------------------------#
+sep(79)
+
+# Show the confidence intervals for the intercept and each virus level.
+cat('Confidence Intervals\n')
+sep(79)
+#-----------------------------------------------------------------------------#
+confint(model)
+#-----------------------------------------------------------------------------#
+sep(79)
+
+# The ANOVA table shows that the virus factor is significant, so we then see
+# which levels are significant with respect to each other using least
+# significant difference.
+cat('Least Significant Difference\n')
+sep(79)
+#-----------------------------------------------------------------------------#
+LSD.test(model, "virus", console=TRUE)
+#-----------------------------------------------------------------------------#
+sep(79)
+
+##Create Post-hoc Bar Graph with LSD letter labels##
+#Save lsd summary
+lsd <- LSD.test(model, "virus", group = TRUE)
+#Save each component in temporary df to use in creating bar graph
+lsd.letters <- as.character(lsd$groups[,3])
+lsd.trt.means <- (lsd$groups[,2])
+lsd.trt.names <- (lsd$groups[,1])
+table1 <- data.frame(lsd.trt.names,lsd.trt.means,lsd.letters)
```

Comment:
`table1` is the exact same data frame as `lsd$groups`, there is no reason to create it like this.